### PR TITLE
Refactoring testing user screens part 2

### PR DESCRIPTION
**Pull request by irina-baeva**
It is just a part of the task on user screens, I am creating PR cuz it is getting big. 
Also there is still blocker -  i118n mock needed

### CODE REVIEW

File: src/api/index.test.js [hunk 1]
```diff
@@ -1,56 +1,234 @@
...
+    expect(article.data.data).toEqual(response);
+  });
+
+  it('should successfully send user data on POST to /login endpoint', async () => {
+    const email = 'test@test.com';
+    const password = '12345678';
+    // given
+    const user = {
+      username: 'Celeste',
+      email,
```
Comment (thdvbr) [inline src/api/index.test.js]: should we have password here ?

File: src/api/index.test.js [hunk 2]
```diff
@@ -1,56 +1,234 @@
...
+      },
+    ];
+    mockFMClient.onGet(`/users/${username}/contacts`).reply(200, response, {
+      headers: { 'auth-token': token },
+    });
+    // when
+    const contacts = await appApiClient.getSosContacts(username, token);
+    // then
+    expect(contacts.data).toEqual(response);
   });
```
Comment (thdvbr) [inline src/api/index.test.js]: looks good ! do we need to test fail cases on the frontend too ?

**Review by thdvbr — APPROVED** [review]

File: babel.config.js [hunk 3]
```diff
@@ -26,6 +26,7 @@ module.exports = (api) => {
             _state: './src/state',
             _services: './src/services',
             _hooks: './src/hooks',
+            helpers: './src/helpers',
```
[the remark this answers is not in the document]
Comment (irina-baeva) [reply babel.config.js:29]: so we need to remove all of them? or i do not understand something. I think we need to have identifier for alias and proper path specification. But what i saw the most used is @

File: src/api/index.test.js (hunk 2 above)
Comment (irina-baeva) [reply src/api/index.test.js]: yes, i think we can send errors and show how FE response to it

File: src/api/index.test.js (hunk 1 above)
Comment (irina-baeva) [reply src/api/index.test.js]: i think no, also do we have password in response? why? we need token only

File: babel.config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (thdvbr) [reply babel.config.js:29]: yes we should eventually remove it.. or unify it somehow :D

File: src/api/index.test.js (hunk 1 above)
Comment (thdvbr) [reply src/api/index.test.js]: i think we have password in the response since we send the whole user document + the token

Comment (thdvbr) [reply src/api/index.test.js]: i think we should only send token no user

Comment (irina-baeva) [reply src/api/index.test.js]: not password for sure, because anyway fe uses only token for auth

**Pull request merged**